tests/e2e/e2e_egress_test.go

```diff
@@ -84,7 +75,7 @@ var _ = OSMDescribe("HTTP and HTTPS Egress",
 						}
 						Td.T.Logf("%s > REST req succeeded: %d", url, result.StatusCode)
 						return true
-					}, 5, 60*time.Second)
+					}, 5, 5*60*time.Second)
```

Comment:
I bumped this timeout because of a DNS bug that is happening on Windows. See https://github.com/envoyproxy/envoy/issues/17795

I can make it conditional based on the clusterOs if you want tighter checks for linux

Reply:
Please open a GH issue to track this, and drop a comment here of the form:
`TODO(#<issue>: lower the timeout once .... is fixed)`.

Also, please make this conditional based on the OS so we don't relax the time for linux, as this could mask timing issues resulting from slow config updates and so on.

Reply:
fixed, see https://github.com/openservicemesh/osm/issues/4027